<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ввод таблицы истинности:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="varSelector">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>2 переменных [a, b]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 переменных [a, b, c]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 переменных [a, b, c, d]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 переменных [a, b, c, d, e]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 переменных [a, b, c, d, e, f]</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="methodSelector">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Преобразовать в СДНФ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Преобразовать в СКНФ</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <widget class="QPushButton" name="buttonConfirm">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>Проверить и добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
